Sources/PointFree/Account/Account.swift

```diff
@@ -767,4 +767,9 @@ private struct AccountData {
   var isSubscriptionOwner: Bool {
     return self.currentUser.id == self.subscriptionOwner?.id
   }
+
+  var isTeamSubscription: Bool {
+    guard let id = stripeSubscription?.plan.id else { return false }
+    return id == Tagged.teamMonthly || id == Tagged.teamYearly
```

Comment:
is it possible to drop the `Tagged` type from these?

Reply:
yes! I ran into some issues with the type checker with an earlier version, and I accidentally left them in :(.